### PR TITLE
complete TDD for /api/articles?:article_id?comment_count= endpoint

### PR DESCRIPTION
**Pull request by HudaifDaif**

### CODE REVIEW

**Review by nickkagame — APPROVED** [review]
Hi Daif, Good work, but think there might be some misunderstanding of the task.  the idea is not for comment count to be a query, but to always be served up on the getArticle endpoint.

**Review by nickkagame — CHANGES_REQUESTED** [review]
Hi Daif, Good work, but think there might be some misunderstanding of the task. the idea is not for comment count to be a query, but to always be served up on the getArticle endpoint.

File: __tests__/api.test.js [hunk 1]
```diff
@@ -76,6 +76,16 @@ describe("/api/articles", () => {
 					expect(body.msg).toBe("Bad Request");
 				});
 		});
+		it.only("200: should respond with a property of comment_count", () => {
+			return request(app)
+				.get("/api/articles/12")
+				.expect(200)
+				.then(({ body }) => {
+					expect(body.article).toMatchObject({
+						comment_count: expect.any(Number),
```
Comment (johnhutcheon) [inline __tests__/api.test.js:85]: You shouldn't need to add this to the previous test. If you remove this line and it's not passing, you may need to use a more flexible matcher 👍

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hey Daif. Nice work! Just see my comment, could we please have a new test in it's own describe block for this ticket please? We shouldn't have to amend the previous test and it should still pass as it was. 👍

File: __tests__/api.test.js (hunk 1 above)
Comment (HudaifDaif) [reply __tests__/api.test.js:85]: Sorry, didn't realise I had a .only in there :facepalm:

**Review by johnhutcheon — APPROVED** [review]
Thanks for making the change Daif! 🙌

**Pull request merged**